qt/src/DisplayRenderer.cc

```diff
@@ -106,3 +110,32 @@ QImage PDFRenderer::render(int page, double resolution) const {
 int PDFRenderer::getNPages() const {
 	return m_document ? m_document->numPages() : 1;
 }
+
+#include <QDebug>
```

Comment:
This looks like a debug leftover which should be removed.

Reply:
Fixed